contracts/facets/GettersFacet.sol

```diff
@@ -311,8 +312,22 @@ contract GettersFacet is IGetters {
         ) {
             allowedWithdrawal = userCollateral + availableYield;
         } else if (collateral.state == LibCollateralStorage.CollateralStates.CycleOngoing) {
-            // Everything above 1.5 X remaining cycles contribution (RCC) can be withdrawn
-            uint minRequiredCollateral = (getRemainingCyclesContributionWei(termId) * 15) / 10; // 1.5 X RCC in wei
+            // uint minRequiredCollateral = (getRemainingCyclesContributionWei(termId) * 15) / 10; // 1.5 X RCC in wei
+
+            uint minRequiredCollateral;
+
+            // Check if the user has paid this cycle
+            if (!fund.paidThisCycle[user]) {
```

Comment:
Wouldn't this be better to put in the getRemainingCyclesContributionWei function?

Reply:
You mean for the `minRequiresCollateral`? I used it if the user have not paid, if the user have already paid the cycle I'm calculating the amount with one cycle less